# This is a copy of the message definition used by the Digital Logger node

string name
bool is_active
